_shared/project/Makefile

```diff
@@ -123,9 +123,9 @@ $(call help,make sure,"make sure that the formatting$(comma) linting and tests a
 sure: python
 sure:
 {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
-	@pyenv exec tox --parallel -qe 'checkformatting,lint,tests,coverage,functests'
+	@pyenv exec tox --parallel -qe 'clean,checkformatting,lint,tests,coverage,functests'
 {% else %}
-	@pyenv exec tox --parallel -qe 'checkformatting,lint,tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage,functests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-functests'
+	@pyenv exec tox --parallel -qe 'clean,checkformatting,lint,tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage,functests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-functests'
```

Comment:
Add the `clean` environment to `make sure` as well.

If you run `make test`, `make format`, `make lint`, etc those commands don't use the `clean` environment (they don't need to since they don't report coverage) so they won't be slowed down by running the `clean` environment.